===================================================================
RCS file: /home2/webcvs/mirror/kdebase/kdesktop/minicli.cpp,v
retrieving revision 1.136.2.8
retrieving revision 1.136.2.9
diff -u -p -r1.136.2.8 -r1.136.2.9
--- kdesktop/minicli.cpp	2003/03/12 09:26:29	1.136.2.8
+++ kdesktop/minicli.cpp	2003/03/14 18:45:41	1.136.2.9
@@ -545,7 +545,6 @@ void Minicli::slotAdvanced()
         m_FocusWidget = focusWidget();
         if( m_FocusWidget )
             m_FocusWidget->setFocus();
-        mpAdvanced->adjustSize();
     }
     else
     {
@@ -555,7 +554,6 @@ void Minicli::slotAdvanced()
             m_FocusWidget->setFocus();
         mpAdvanced->setMaximumSize(0, 0);
         mpAdvanced->setEnabled( false );
-        mpAdvanced->adjustSize();
     }
 }

